engine/src/org/pentaho/di/trans/TransMeta.java

```diff
@@ -2878,7 +2880,7 @@ public void loadXML( Node transnode, String fname, IMetaStore metaStore, Reposit
             }
           }
         }
-        setPrivateTransformationDatabases( privateTransformationDatabases );
+        setPrivateDatabases( privateTransformationDatabases );
```

Comment:
@mattyb149. your remark makes sense, but I suppose it is not good to keep the old method, as it is useless.

The previous fix was merged recently, it is not included into any published version. Therefore, it is just a part of SNAPSHOT. And no one guarantees snapshot steadiness, especially, if it is not an interface. That's my vision. Do you still disagree?